app/backend/error.py

```diff
@@ -9,10 +9,14 @@
 """
 ERROR_MESSAGE_FILTER = """Your message contains content that was flagged by the OpenAI content filter."""
 
+ERROR_MESSAGE_LENGTH = """Your message exceeded the context length limit for this OpenAI model. Please shorten your message or retrieve less sources from the search index."""
```

Comment:
Should we tell the user how to do "retrieve less sources from the search index"?

Reply:
Rewording to "Please shorten your message or change your settings to retrieve fewer search results."

I don't want to get too exact in case customer changes UI but that feels both fairly generic but also gives a good hint in terms of the settings.